eos/rare-b-decays/bs-to-phi-ll.cc

```diff
@@ -91,7 +91,7 @@ namespace eos
         {
             Context ctx("When constructing Bs->Phill observables");
 
-            std::string tag = o.get("tag"_ok, "");
+            std::string tag = o.get("tag"_ok, "GvDV2020");
```

Comment:
Please keep this default value empty. We don't want to provide a default value here.